bookwyrm/templates/feed/layout.html

```diff
@@ -1,9 +1,26 @@
 {% extends 'layout.html' %}
 {% load i18n %}
+{% load utilities %}
 
 {% block title %}{% trans "Updates" %}{% endblock %}
 
 {% block content %}
+{% if user.moved_to %}
+<div class="container my-6">
+    <div class="notification is-info has-text-centered">
+        <p>
+            {% trans "You have have moved to" %}
```

Comment:
It would be better to include the link in a blocktrans, so that languages with different word order can place the link in different places